lib/MusicBrainz/Server/Data/Utils.pm

```diff
@@ -326,6 +326,8 @@ sub collapse_whitespace {
 sub sanitize {
     my $t = shift;
 
+    return '' if not defined $t;
```

Comment:
Maybe we can call this $t something a bit more descriptive while we're touching it? $text or whatever it wants to be?

Reply:
I would certainly avoid adding more one-letter variables, but this one is already here and doesn’t seem to impair readability as `sanitize` is fairly short and obviously about text.